components/admin_console/group_settings/group_details/group_teams_and_channels_row.test.jsx

```diff
@@ -84,6 +84,6 @@ describe('components/admin_console/group_settings/group_details/GroupTeamsAndCha
             />
         );
         wrapper.find('.btn-link').simulate('click');
-        expect(onRemoveItem).toBeCalledWith('xxxxxxxxxxxxxxxxxxxxxxxxxx', 'public-team');
+        expect(wrapper.instance().state.showConfirmationModal).toEqual(true);
```

Comment:
Here you are replacing the previous tested case with your new tested behavior, I would expect to have the previous one too, or at least, the "continuation of the test", something like `wrapper.find('.class-for-the-confirm-button').simulate('clic');` and `expect(onRemoveItem).toBeCalledWith('xxxxxxxxxxxxxxxxxxxxxxxxxx', 'public-team');`

Reply:
Let me know your thoughts on this update (keeping in mind that the shallow render no longer allows access to the confirm modal confirm button).

Reply:
Yes, that works for me :)